lib/wings.rb

```diff
@@ -49,6 +45,28 @@ module Wings; end
 [Hyrax::CustomQueries::Wings,
  Hyrax::CustomQueries::Navigators::ChildCollectionsNavigator,
  Hyrax::CustomQueries::Navigators::ChildFilesetsNavigator,
- Hyrax::CustomQueries::Navigators::ChildWorksNavigator].each do |query_handler|
+ Hyrax::CustomQueries::Navigators::ChildWorksNavigator,
+ Hyrax::CustomQueries::FindAccessControl].each do |query_handler|
   Valkyrie.config.metadata_adapter.query_service.custom_queries.register_query_handler(query_handler)
 end
+
+Hydra::AccessControl.define_method(:valkyrie_resource) do
+  attrs = attributes.symbolize_keys
+  attrs[:new_record]  = new_record?
+  attrs[:created_at]  = create_date
+  attrs[:updated_at]  = modified_date
+
+  attrs[:permissions] = permissions.map do |permission|
+    agent = permission.type == 'group' ? "group/#{permission.agent_name}" : permission.agent_name
```

Comment:
Would it make sense to consider having the `group/` prefix be a constant? The usage seems pretty much isolated to private methods in `PermissionManager` and  `PermissionValue#result`. Just a thought.

Reply:
yeah, i think it would be a good idea. i didn't pursue it during development because it wasn't clear to me where that constant should be namespaced.

i think it's still not clear, but this is all static enough now that we could probably just pick a place and have it be fine. ideas?

Reply:
I wasn't sure either.. I'm tempted to say `Hyrax::Group.prefix`. At least to me that reads fairly directly and seems in context. That's my gut reaction, I suppose.